cosmovisor/cmd/cosmovisor/cmd/version.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version number of Cosmovisor and the application",
+	Long:  `Prints the version number of Comovisor and the underlying application`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		// TODO - get actual version number
```

Comment:
Can you also make a variable for `Version`
 
```
var  Version = ""
```
Once #10110 is merged, we can use ldflags to set Version

add `ldflags` in cosmovisor/.goreleaser
 ```
    ldflags:
      - -X github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor/cmd/Version={{ replace .Version "cosmovisor/" "" }}
   ```

Reply:
Thanks. Added the `Version` var